.appveyor.yml

```diff
@@ -37,10 +58,15 @@ build_script:
 after_build:
 - ps:  fsutil behavior set disablelastaccess 1 # Disable Access time feature on Windows (better performance)
 - ps:  clcache -z
+#- 7z a bitcoin-%APPVEYOR_BUILD_VERSION%.zip %APPVEYOR_BUILD_FOLDER%\build_msvc\%platform%\%configuration%\*.exe
```

Comment:
What are the comments in this file?

Reply:
The comments are there for devs who would like to get artifacts from the appveyor build. The official decision was not to generate binaries from the Bitcoin Core build since it's not trusted. But it's still useful to provide the commented steps for devs who make a deliberate decision to enable artifacts.

Reply:
Ah, so you could keep this 7z command uncommented, but leave the artifacts below commented?

no strong opinion, though